src/dotnet/APIView/APIViewWeb/Helpers/PageModelHelpers.cs

```diff
@@ -334,11 +334,12 @@ public static int ComputeActiveConversationsInActiveRevision(CodeLine[] lines, R
                 Review = review,
                 Navigation = activeRevisionRenderableCodeFile.CodeFile.Navigation,
                 codeLines = codeLines,
-                APIRevisions = revisions.GroupBy(r => r.APIRevisionType).ToDictionary(r => r.Key.ToString(), r => r.ToList()),
-                ActiveRevision = activeRevision,
-                DiffRevision = diffRevision,
+                APIRevisionsGrouped = revisions.GroupBy(r => r.APIRevisionType).ToDictionary(r => r.Key.ToString(), r => r.ToList()),
+                ActiveAPIRevision = activeRevision,
+                LatestApprovedAPIRevision = revisions.Where(r => r.IsApproved).OrderByDescending(r => r.ChangeHistory.Last(ch => ch.ChangeAction == APIRevisionChangeAction.Approved).ChangedOn).First(),
```

Comment:
it doesn't look like this is used anywhere. is the idea to consume it later?

Reply:
Right. I started to implement feature requested by ted, which was a way to get to the LatestApprovedRevision. I can remove it for now.